app/lib/actions/katello/organization/manifest_delete.rb

```diff
@@ -16,10 +16,15 @@ def plan(organization)
             repositories.each do |repo|
               plan_action(Katello::Repository::RefreshRepository, repo)
             end
-            plan_self
+            plan_self(:organization_name => organization.name)
```

Comment:
I don't see why this would be an issue for lookup, but is there a reason to use name over id here?

Reply:
Just consistency with `manifest_import.rb`. In the [previous PR](https://github.com/Katello/katello/pull/10999) the `plan_self` was already there with `name` so I just did it the same way here.

Reply:
Okay, that's totally reasonable. This should be fine 👍